README.md

```diff
@@ -926,212 +573,629 @@ Response:
+In a future release of this SDK we also plan to support an expiration period associated with the signed tokens making it very powerful when you want to detokenize data from your front end application. 
 
-To delete data from the vault, use the `delete(records, options?)` method of the Skyflow client. The `records` parameter takes an array of records to delete in the following format. The `options` parameter is optional and takes an object of deletion parameters. Currently, there are no supported deletion parameters.
+When the context aware bearer tokens along with the signed data tokens are sent to the detokenize endpoint, Skyflow’s governance engine performs the following checks: 
+ - Validate the signature of the bearer token
+ - Bearer token has not expired
+ - Validate the signature of the signed data token
+ - Signed data token has not expired
+ - ctx value in the bearer token matches the ctx value in the signed token
+ - Service account has permissions to detokenize data 
 
-Call schema:
-```js
-const deleteInput = {
-  records: [
-    {
-      id: "<SKYFLOW_ID_1>", // skyflow id of the record to delete
-      table: "<TABLE_NAME>" // Table from which the record is to be deleted
-    },
-    {
-      // ...additional records here
-    },
-  ]
-};
+Only if these conditions are met, will the detokenize request be successful 
 
-const options = {
-  // Optional
-}
-```
+The data tokens are passed as part of the `options` in the `generateSignedDataTokens(filepath, options)` function, which takes the service account credentials file path and an options object for token generation. Alternatively, you can send the entire service account credentials as string, by using the `generateSignedDataTokensFromCreds(credentials, options)` function.
 
-[Example](https://github.com/skyflowapi/skyflow-node/blob/master/samples/vault-api/Delete.ts) to delete by ID using `skyflow_ids`
+<details>
+<summary>
+Example using a service account credentials file path
+</summary>
+
+[Example using a service account credentials file path:](https://github.com/skyflowapi/skyflow-node/blob/master/samples/service-account/SignedTokenGenerationExample.ts)
 
 ```js
-const deleteInput = {
-  records: [
-    {
-      id: "29ebda8d-5272-4063-af58-15cc674e332b",
-      table: "cards",
-    },
-    {
-      id: "d5f4b926-7b1a-41df-8fac-7950d2cbd923",
-      table: "cards",
-    }
-  ],
-};
+import { generateSignedDataTokens,isExpired } from 'skyflow-node';
 
-const options = {};
+let filepath = 'CREDENTIALS_FILE_PATH';
+let bearerToken = '';
 
-const response = skyflowClient.delete(deleteInput, options);
```

Comment:
This example was inconsistent with the rest, which were using `client` not `skyflowClient`. I changed them all to be `vault`.